src/main/java/com/icthh/xm/uaa/domain/UserLoginType.java

```diff
@@ -35,4 +36,10 @@ public String getValue() {
         return Optional.empty();
     }
 
+    public static UserLoginType fromString(String value) {
+        return Arrays.stream(UserLoginType.values())
```

Comment:
Here is an inefficient enum values extraction because values() method always creates an array. 
Propose to cache enym values in static variable like:
```
    private final static Set<UserLoginType> VALUES = Set.of(UserLoginType.values());
```
and then:
```
 return VALUES.stream()
            .filter( ... 
```